java/client/src/main/java/glide/api/commands/servermodules/Json.java

```diff
@@ -411,6 +413,85 @@ public static CompletableFuture<GlideString> get(
                 new ArgsBuilder().add(gs(JSON_GET)).add(key).add(options.toArgs()).add(paths).toArray());
     }
 
+    /**
+     * Retrieves the JSON values at the specified <code>path</code> stored at multiple <code>keys
+     * </code>.
+     *
+     * @apiNote When in cluster mode, if keys in <code>keys</code> map to different hash slots, the
+     *     command will be split across these slots and executed separately for each. This means the
+     *     command is atomic only at the slot level. If one or more slot-specific requests fail, the
+     *     entire call will return the first encountered error, even though some requests may have
+     *     succeeded while others did not. If this behavior impacts your application logic, consider
+     *     splitting the request into sub-requests per slot to ensure atomicity.
+     * @param client The client to execute the command.
+     * @param keys The keys of the JSON documents.
+     * @param path The path within the JSON documents.
+     * @return An array with requested values for each key.
+     *     <ul>
+     *       <li>For JSONPath (path starts with <code>$</code>): Returns a stringified JSON list
+     *           replies for every possible path, or a string representation of an empty array, if
+     *           path doesn't exist.
+     *       <li>For legacy path (path doesn't start with <code>$</code>): Returns a string
+     *           representation of the value in <code>path</code>. If <code>path</code> doesn't exist,
+     *           the corresponding array element will be <code>null</code>.
+     *     </ul>
+     *     If a <code>key</code> doesn't exist, the corresponding array element will be <code>null
+     *     </code>.
+     * @example
+     *     <pre>{@code
+     * Json.set(client, "doc1", "$", "{\"a\": 1, \"b\": [\"one\", \"two\"]}").get();
+     * Json.set(client, "doc2", "$", "{\"a\": 1, \"c\": false}").get();
+     * var res = Json.mget(client, new String[] { "doc1", "doc2", "doc3" }, "$.c").get();
+     * assert Arrays.equals(res, new String[] { "[]", "[false]", null });
+     * }</pre>
+     */
+    public static CompletableFuture<String[]> mget(
+            @NonNull BaseClient client, @NonNull String[] keys, @NonNull String path) {
+        return Json.<Object[]>executeCommand(
```

Comment:
add a check here for keys is not empty - otherwise, we should return RequestError in the completablefuture